content/tutorials/xgboost_deployment.md

```diff
@@ -4,79 +4,57 @@ date= 2018-03-12T17:08:30-07:00
 description = "Guide to deploy XGBoost Models."
 draft= false
 +++
-## A guide to installing packages to model containers
+## A guide to deploy XGBoost Models
 {{% notice warning %}}
 This tutorial will assume that you have already installed `clipper_admin` and its dependencies.
 {{% /notice %}}
 
-In order to keep the base image's light, containers limit the packages they come with default; however, Clipper also supports the installation of additional packages onto model containers using pip. Packages such as XGBoost can be installed and deployed in this way.
+In order to keep the base images small, containers limit the packages they come with default; however, Clipper also supports the installation of additional packages onto model containers using pip. Packages such as XGBoost can be installed and deployed in this way.
 
-First, we open a python interactive shell.
+First, we open a python interactive shell, and start clipper.
 ```sh
-$ python
-# Now, we start clipper.
 >>> import logging, xgboost as xgb, numpy as np
 >>> from clipper_admin import ClipperConnection, DockerContainerManager
 >>> cl = ClipperConnection(DockerContainerManager())
```

Comment:
Delete the `>>>` (so readers can directly copy and paste) from all of the code blocks